dashboard/model_layer.py

```diff
@@ -158,7 +158,8 @@ def enrich_session_info(se):
     se.data_transferred = se.client_bytes_sent + se.client_bytes_received
     se.started = humanize.naturaltime((datetime.utcnow() - se.created_at).total_seconds())
     se.status = 'Ongoing' if ((se.node_updated_at or se.client_updated_at) >= datetime.utcnow() - timedelta(minutes=NODE_AVAILABILITY_TIMEOUT)) else 'Completed'
-
+    se.shortened_node_key = (se.node_key[:6] + '..' + se.node_key[-4:]) \
+        if len(se.node_key) == models.IDENTITY_LENGTH_LIMIT else se.node_key
```

Comment:
This is making assumption that all node keys are at least 10 digits long - if for some unexpected reason we receive empty node_key, this will cause Internal Server Error. The less assumption we make - the safer we are. Can we make don't cause 500 error with empty `node_key`?

Reply:
fixed